vpr/src/base/vpr_context.h

```diff
@@ -369,6 +370,17 @@ struct RoutingContext : public Context {
         cached_router_lookahead_;
 };
 
+/**
+ * @brief State relating to VPR's floorplanning constraints
+ *
+ * This should contain only data structures related to constraining blocks
+ * to certain regions on the chip.
+ */
```

Comment:
comment what the two data members are (vpr constraints can be brief, clb_constraints should have some more explanation)

Reply:
will be a bit faster in a vtr::vector because the IDs will be dense (every cluster will be in here, even if it just an empty PR)
there is a vtr::vector and vtr::vector_map - vector is probably enough in this case.

Reply:
vtr::vector is better for this case